.github/workflows/manageiq_cross_repo.yaml

```diff
@@ -19,6 +19,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
+      matrix:
+        test-repo: ${{ fromJson(inputs.test-repo) }}
```

Comment:
NOTE arrays are not supported (only string, boolean, number) so you have to pass a string containing a json array then parse that on this side

Reply:
Example: https://github.com/ManageIQ/manageiq-cross_repo-tests/pull/582/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR9-R10